lib/config.js

```diff
@@ -259,6 +260,16 @@ internals.server = Validate.object({
     })
         .unknown()
         .default(),
+    logger: Validate.object({
+        additionalFields: Validate.object(),
+        level: Validate.valid('emergency', 'alert', 'critical', 'error', 'warning', 'notice', 'info', 'debug').default('info'),
+        stderr: Validate.object().optional().instance(Stream.Writable),
```

Comment:
Writable does not really support `instanceof` check. Eg. any stream inherited from `Duplex` will fail this check, as it inherits from `Readable`. `Stream` will be the safe check.